hw/ip/rom_ctrl/dv/cov/rom_ctrl_cov_excl.el

```diff
@@ -146,6 +146,135 @@ Condition 37 "2164803938" "(tl_i_int.a_valid & reqfifo_wready & ((~error_interna
 // u_sram_byte. But this makes req_o false and hence sram_ack false.
 Condition 43 "2041272341" "(sram_ack & ((~we_o))) 1 -1" (2 "10")
 
+// The case 1101 is non-occuring for the conditional statement below.
+//
+// One reason is that the depth of the fifos is 2. When we write something in the fifo, we read the
+// same thing on the next cycle and update the wptr and rptr respectively. It can happen if we
+// delay reads and make them happen after writing to both of the location of the fifos. This will
+// lead to increment only wptr. But this can't happen for u_sramreqfifo because when we write into
+// the fifo we will get rvalid_o in u_sramreqfifo which depends on empty and under_rst. For not
+// having sramreqfifo_rready when we write to the fifo is not possible as it depends on
+// reqfifo_rvalid and we can't hope for not having a reqfifo_rvalid if there was a request to
+// write something in u_sramreqfifo on the last cycle.
+CHECKSUM: "1909897872 4057018282"
+INSTANCE: tb.dut.u_tl_adapter_rom
+Condition 35 "3680494467" "((gnt_i | missed_err_gnt_q) & reqfifo_wready & sramreqfifo_wready
+                            & sramreqaddrfifo_wready) 1 -1" (3 "1101")
+
+// The case 011 for the conditional statements below is non-occuring.
+//
+// When rvalid_o=0, the fifo is empty and it is seen as reqfifo_rvalid in tlul_adapter_sram.sv.
+// rready_i comes in as input from tlul_adapter_sram. If rready_i=1, it means d_valid is true which
+// goes to u_sram_byte as one of the port of tl_o_int and d_ready port of tl_i_int is true which
+// comes out from u_sram_byte. But d_valid to be true requires reqfifo_rvalid true.
```

Comment:
I had to think a bit to convince myself of why this was true and I think it can be reworded a bit to make it clearer?
```
    // This item is impossible because it is rready_i -> rready_o for a u_reqfifo instance in any
    // tlul_adapter_sram. This is because rready_i is driven by reqfifo_rready, which is d_ack. This
    // is only true when tl_o_int.d_valid is true, which means the ROM is responding to a request.
    // That request will have been pushed into u_reqfifo as it was sent out, so the FIFO is not
    // empty and rready_o must be true.
```

Does the reasoning sound right to you?